Sources/UINotification.swift

```diff
@@ -57,11 +57,15 @@ public final class UINotification: Equatable {
         case custom(height: CGFloat)
 
         internal var value: CGFloat {
+            guard let statusBarManager = UIApplication.shared.windows.first(where: \.isKeyWindow)?.windowScene?.statusBarManager else {
```

Comment:
<!--
  2 failure:  'statusBarManager..., 'windowScene' is ...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: Sources/UINotification.swift;
  Line: 60;
-->
- :no_entry_sign: 'statusBarManager' is only available in iOS 13.0 or newer
- :no_entry_sign: 'windowScene' is only available in iOS 13.0 or newer